storage/mysql/tree_storage.go

```diff
@@ -190,18 +205,28 @@ func (t *treeTX) getSubtrees(ctx context.Context, treeRevision int64, ids [][]by
 		}
 	}()
 
-	args := make([]interface{}, 0, len(ids)+3)
+	var args []interface{}
+	if t.subtreeRevs {
+		args = make([]interface{}, 0, len(ids)+3)
+		// populate args with ids.
+		for _, id := range ids {
+			klog.V(4).Infof("  id: %x", id)
```

Comment:
nit: It might be worth changing this (and below) to be `if klog.V(4) { klog.Infof("...") }` while you're here to avoid creating and discarding the formatted log string for every subtree read?

Reply:
Ah, nice - I didn't know `klog` did that internally - thanks!